multinet/__init__.py

```diff
@@ -53,6 +57,8 @@ def create_app(config: Optional[MutableMapping] = None) -> Flask:
     app.register_blueprint(uploaders.nested_json.bp, url_prefix="/api/nested_json")
     app.register_blueprint(uploaders.d3_json.bp, url_prefix="/api/d3_json")
 
+    app.register_blueprint(uploaders.multipart_upload.bp, url_prefix="/api/uploads")
```

Comment:
I don't think this should go in the `uploaders` package but I'm not sure. @AlmightyYakob, thoughts?

Reply:
Yeah I agree, it should probably have its own `upload` blueprint, but I'd be okay dealing with it in a later PR.